test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -4634,6 +4634,33 @@ def forward(self, x):
                           input_names=["x"],
                           dynamic_axes={"x": {0: "batch_size", 1: "dims"}})
 
+    @skipIfUnsupportedMinOpsetVersion(11)
+    @disableScriptTest()
```

Comment:
nit: would be nice to comment the reason why script is disabled

Reply:
Enabled script tests now